migrate/011_add_ipv4.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:sshable) do
+      set_column_allow_null :host
+    end
+    alter_table(:vm_host) do
+      add_column :provider, :text, collate: '"C"'
```

Comment:
I think splitting the location to provider and region/location makes more sense. Especially because provider can be used to filter other stuff in our code such as which endpoint to hit as it is here.